kernel/lib/bitmap.cpp

```diff
@@ -17,7 +17,9 @@ size_t bitmap_find_first_range_clear(bitmap_t *bitmap, size_t size, size_t count
 #ifdef TESTING
         printf("i = %d, idx = 0x%08x, ofst = 0x%08x, check_lo = 0x%08x, "
                "check_hi = 0x%08x, check = 0x%08x, masked = 0x%08x\n",
-                i, idx, ofst, check_lo, check_hi, check, masked); 
+                i, idx, ofst, check_lo, check_hi, check, masked);
+#else
+        (void)masked;
```

Comment:
Just include the declaration and assignment inside the `ifdef`.

Reply:
Done.